src/components/BuildLogPage/BuildLogPage.tsx

```diff
@@ -26,22 +26,33 @@ export const BuildLogPage = () => {
 
   const logData = useMemo(() => serviceContainerBuildLog.data?.split(/[\r\n]/) || [], [serviceContainerBuildLog.data]);
 
-  const [logBuffer, addLogLines] = useDataBuffer(750, timestampHiglighter);
+  const [logBuffer, addLogLines] = useDataBuffer(timestampHiglighter);
 
   useEffect(() => {
     if (!isBuilding) {
       serviceContainerBuildLogRunner({ serviceData: { id: buildId } });
     }
   }, [serviceContainerBuildLogRunner, buildId, isBuilding]);
 
+  const filters = useMemo(
```

Comment:
I have to put filters into useMemo because otherwise the rerender of the filter will cause useBifrostWebSocketEffect created again and again